scripts/symbol_check/readme.md

```diff
@@ -0,0 +1,88 @@
+# Symbol Check
+
+This tool will check the symbols used by user's Mystikos application, and report any unsupported glibc symbols. This tool will run as part of `scripts/appbuilder`, thus only checks Mystikos application that is based on a Docker image.
+
+> For example, if glibc has a symbol `do_something_in_glibc` and Mystikos's CRT lacks the support for it, and at the same time an application (for example /bin/main) uses this symbol. When user is generating `appdir` using `scripts/appbuilder`, this tool should print warning message about the missing support for `do_something_in_glibc` for program `appdir/bin/main`.
```

Comment:
Suggest to be more specific with:

`(for example /bin/main) uses this symbol, either in the executably directly or indirectly in one of its dependent libraries,...`

Reply:
We should encourage the users to create an issue when a missing symbol is found. I know it's present in the warning message. But here is a more visible place.